packages/screens/RiotGame/component/RiotGameHeader.tsx

```diff
@@ -35,7 +35,9 @@ import { PickByValue } from "../../../utils/types/helper";
 type MenuItem = {
   id: string;
   name: string;
-  route?: keyof PickByValue<RootStackParamList, undefined>;
+  route?:
+    | keyof PickByValue<RootStackParamList, undefined>
+    | "RiotGameMarketplace";
```

Comment:
why do you need this? the `RiotGameMarketplace` should already be in the `keyof PickByValue<RootStackParamList, undefined>` if the navigation is configured correctly

Reply:
We have to support RiotGameMarketplace with and without ```{ collectionId?: string }``` params.
Later on this file we declare a MenuItem like this 
```
  {
    id: "marketplace",
    name: "Marketplace",
    route: "RiotGameMarketplace",
    iconSVG: cartSVG,
  },
```
So I don't file other way than that to support both cases. Might be you have other idea ?